manifest: samples: remove openthread 1.2 overlay from sample and doc (#6542)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

* Added ``EXPERIMENTAL`` select in Kconfig that informs that Matter support is experimental.

Thread
------

> * Added support for Thread 1.2 for more info please refer to :ref:`thread_ug_thread_specification_options` and :ref:`thread_ug_feature_sets`.

Reviewer: ```suggestion
* Added support for Thread 1.2, refer to :ref:`thread_ug_thread_specification_options` and :ref:`thread_ug_feature_sets` for more information.
```

Author: fixed

> * Added support for Thread 1.2, refer to :ref:`thread_ug_thread_specification_options` and :ref:`thread_ug_feature_sets` for more information.

Reviewer 2: I'm not quite happy with this wording since we already had Thread 1.2 support in 1.6.0.

I believe the emphasis should be made in how the way Thread 1.2 support is enabled has changed.

Author: fixed

> * Thread 1.2 enabling method has changed, refer to :ref:`thread_ug_thread_specification_options` and :ref:`thread_ug_feature_sets` for more information.

Reviewer: ```suggestion
* Changed how Thread 1.2 is enabled, refer to :ref:`thread_ug_thread_specification_options` and :ref:`thread_ug_feature_sets` for more information.
```
To have it in the same tense as the other entries.

Author: fixed

  * HTTP update samples now set the modem in the power off mode after the firmware image download completes. This avoids ungraceful disconnects from the network upon pressing the reset button on the kit.

Thread samples
--------------

> * :ref:`_ot_cli_sample` sample:

Reviewer: ```suggestion
* :ref:`ot_cli_sample` sample:
```
Breaks the doc build

> * :ref:`_ot_cli_sample`
> 
>   * Removed `overlay-thread_1_2.conf` Thread 1.2 will be now supported as described in :ref:`thread_ug_thread_specification_options`.

Reviewer: ```suggestion
* :ref:`_ot_cli_sample` sample:

  * Removed `overlay-thread_1_2.conf` as Thread 1.2 is now supported as described in :ref:`thread_ug_thread_specification_options`.
```

Author: fixed.
